server/server.js

```diff
@@ -0,0 +1,313 @@
+import "@babel/polyfill";
+import createShopifyAuth, { verifyRequest } from "@shopify/koa-shopify-auth";
+import Shopify from "@shopify/shopify-api";
+import dotenv from "dotenv";
+import "isomorphic-fetch";
+import Koa from "koa";
+import Router from "koa-router";
+import next from "next";
+import Shop from "../models/shop.model";
+import User from "../models/user.model";
+import {
+	deleteCallback,
+	loadCallback,
+	storeCallback
+} from "../utilities/redis-store";
+
+const _ = require("lodash");
+const mongoose = require("mongoose");
+const bodyParser = require("koa-bodyparser");
+const cors = require("@koa/cors");
+const fs = require("fs");
+const axios = require("axios");
+dotenv.config();
+
+mongoose
+	.connect(process.env.MONGODB_URL, {
```

Comment:
use of the custom mongo storage is against what was agreed - use app data instead, we don't need to have a storage at all

Reply:
We need mongo storage to store shop access token. It's required to call App-data metafield API

Reference: https://shopify.dev/docs/apps/custom-data/metafields/app-data